autorest/adal/token.go

```diff
@@ -709,36 +713,51 @@ func retry(sender Sender, req *http.Request) (resp *http.Response, err error) {
 		http.StatusNotExtended,
 		http.StatusNetworkAuthenticationRequired)
 
+	// see https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/how-to-use-vm-token#retry-guidance
+
+	const maxAttempts int = 5
+	const deltaDelay int = 2          // seconds
+	const maxDelay time.Duration = 60 // seconds
```

Comment:
Nit: I'd declare this as `const maxDelay = 60 * time.Second`

Reply:
Fixed.